src/server/routes/api/document/indra.js

```diff
@@ -133,17 +137,51 @@ const getDocuments = ( templates, queryArticle ) => {
         headers: { 'Content-Type': 'application/json' }
       })
       .then( res => res.json() )
-      .then( semanticScores => ( { pubmeds, semanticScores } ) );
+      .then( semanticScores => ( { articles, semanticScores } ) );
+    };
+
+    const getPubTime = article => {
+        let dateJs = article.Journal.JournalIssue.PubDate;
+        let { Day: day, Year: year, Month: month, MedlineDate: medlineDate } = dateJs;
+
+        if ( medlineDate ) {
+          return new Date( medlineDate );
```

Comment:
There's a strong probability that this will throw.

Reply:
Okay, I have returned the minimum possible integer in such cases.